crates/tests/src/lib.rs

```diff
@@ -183,7 +183,7 @@ impl MadaraCmdBuilder {
         let project_root = PathBuf::from(cargo_toml_path.trim()).parent().unwrap().to_path_buf();
 
         env::set_current_dir(&project_root).expect("Failed to set working directory");
-        let target_bin = option_env!("COVERAGE_BIN").unwrap_or("./target/debug/madara");
+        let target_bin = option_env!("COVERAGE_BIN").expect("COVERAGE_BIN env");
```

Comment:
just use env! instead of option_env! in this case no?
also, why do you need that? if we're setting the current dir to project root it should be fine right?